Cloudpebble : Cannot find module 'message_keys'

[PHONE] pebble-app.js:?: JavaScript Error:
Error: Cannot find module 'message_keys'
    at Object.loader.require (loader.js:66:11)
    at _require.require (loader.js:54:48)
    at o (node_modules/pebble-clay/index.js:2:419)
    at node_modules/pebble-clay/index.js:2:610
    at Object.pebble-clay../package.json (node_modules/pebble-clay/index.js:15:1563)
    at o (node_modules/pebble-clay/index.js:2:559)
    at e (node_modules/pebble-clay/index.js:2:726)
    at node_modules/pebble-clay/index.js:2:743
    at c (node_modules/pebble-clay/index.js:2:85)
    at Object.loader (node_modules/pebble-clay/index.js:2:282)

--- a/README.md
+++ b/README.md
@@ -25,7 +25,7 @@ If you would like to contribute to Clay, check out the [contributing guide.](CON
 
 # Getting Started (CloudPebble)
 
-1. Ensure `JS Handling` is set to `CommonJS-style` in your project settings.
+1. Ensure `Configurable` is checked and `JS Handling` is set to `CommonJS-style` in your project settings.
 2. Under `Dependencies` in the project navigation, enter `pebble-clay` as the `Package Name` and `^1.0.0` for the `Version`. You may use any specific version you like, however using `^1.0.0` will ensure you receive all minor version updates.
 4. Create a JavaScript file called `config.js` with the following content. This will act as your config's root array element, from which the rest of the page is built up:
 

